app/models/taging.rb

```diff
@@ -0,0 +1,13 @@
+class Taging < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :task
+
+
+  def self.add_tag(user_id,task_id)
+    Taging.create(user_id: user_id,task_id: task_id)
+  end
+
+  def remove_tag
```

Comment:
@rakesh-1010 You should delete tag using `task.tag.destroy` rather than using this class method.